src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/HarborSelection.tsx

```diff
@@ -18,6 +18,9 @@ type StopPlaceSelectionProps = {
   style?: StyleProp<ViewStyle>;
 };
 
+const FROM = 'from';
+const TO = 'to';
```

Comment:
In situations like this, if we need a type, these should rather be `type fromOrTo = 'from' | 'to'` or maybe `type direction = 'from' | 'to'`.

In this case where it is used only one placed, I think I would just inline it in the props to the component, like `fromOrTo: 'from' | 'to'`.